governance/chair-proposal-process.md

```diff
@@ -0,0 +1,148 @@
+# Environmental Sustainability Leadership Team
+
+The leadership team of the WG are servant leads. They are an enabler for the group and support the creation and success of the different activities. They are the driver for the group for not losing the higher targets out of sight and taking decisions when no consensus can be found.
+
+## Team Structure
+
+The leadership team is conducted of:
+* Three Co-Chairs
+* Three Tech Leads
+
+As WG we strive to be a gender, company and geolocation, diverse group. 
+Therefore, the whole leadership team is not allowed to have more than two people from the same organization. This also includes companies that belong through majority ownership to another company. 
+
+
+# Proposal Process
+
+## Environmental Sustainability WG Chair proposal process
+
+1) Environmental Sustainability WG Co-chairs operate as a team. The Co-Chair team seeks to maintain full coverage of the leadership capabilities across the key qualities of industry experience, hands-on cloud-native and sustainability experience, as well as administrative experience needed to run a Environmental Sustainability WG. 
+2) If a clear successor for the exiting Co-Chair exists based on the merit of their contributions ("plant trees, carry water") and relevant experience that will benefit the WG, based on the WG's collective experience over time, the exiting Co-chair may nominate a successor.
+3) The exiting Co-Chair should propose the desired successor to the other Co-Chairs. If there are any objections, the Co-Chair will have the opportunity to propose another worthy candidate. If Environmental Sustainability WG Chairs, Tech Leads and TOC Liaisons are unable to put forward a candidate, then a selection process will need to be agreed upon and executed.
```

Comment:
Another case of WG

Reply:
I would strike this as well as this is likely to be part of TAG governance from the TOC